packages/donor/src/screens/questionnaire/questions/QuestionsHadasaEinKerem.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from "react";
+import Popup from "../../../components/basic/Popup";
+import WhatsappIcon from "../../../assets/images/whatsup-color-big.svg";
+import { Question, YesNoNotRelevantOptions, YesNoOptions } from "./Question";
+import { LinkUtils } from "@zm-blood-components/common";
+import { QuestionsProps } from "./QuestionnaireQuestions";
+
+export default function QuestionsHadasaEinKerem({
+  setAreAllAnswersCorrect,
+  goToHomePage,
+}: QuestionsProps) {
+  const [isWeightValid, setIsWeightValid] =
+    React.useState<boolean | undefined>(undefined);
+  const [isSurgeryValid, setIsSurgeryValid] =
+    React.useState<boolean | undefined>(undefined);
+  const [isRightAge, setIsRightAge] =
+    React.useState<boolean | undefined>(undefined);
+  const [wasPregnant, setWasPregnantEver] =
+    React.useState<string | undefined>(undefined);
+  const [isTattooValid, setIsTattooValid] =
+    React.useState<boolean | undefined>(undefined);
+
+  useEffect(() => {
+    const res =
+      (isWeightValid &&
+        isSurgeryValid === false &&
+        isTattooValid === false &&
+        isRightAge &&
+        wasPregnant !== "yes") ||
+      false;
+
+    setAreAllAnswersCorrect(res);
+  }, [
+    isWeightValid,
+    isSurgeryValid,
+    isRightAge,
+    wasPregnant,
+    isTattooValid,
+    setAreAllAnswersCorrect,
+  ]);
+
+  const isWrongAnswerChosen =
+    isWeightValid === false ||
+    isSurgeryValid ||
+    isTattooValid ||
+    isRightAge === false ||
+    wasPregnant === "yes";
+
+  return (
```

Comment:
I'm guessing these questions aren't special to Hadassah... wouldn't we want to create a common module with them ?

Reply:
Every hospital has their special questions. Most are common but each hospital has their own changes.

How would you suggest doing that?